cmd/interfaces.go

```diff
@@ -17,7 +17,7 @@ type execRunner interface {
 
 type envExecRunner interface {
 	execRunner
-	Env(e []string)
+	SetEnv(e []string)
```

Comment:
This change is not related to the scope of the pull request which is "Move mock capabilities ..."